packages/api/endpoints/launchpadSaml.js

```diff
@@ -162,7 +162,7 @@ const prepareSamlProviders = async () => {
 
   const idpOptions = {
     sso_login_url: process.env.IDP_LOGIN,
-    sso_logout_url: undefined,
+    sso_logout_url: null,
```

Comment:
If that value really is required, specifying `null` here is going to be just as broken as setting it to `undefined` or not setting it at all. Unless we have a valid value, then I vote we just remove it to avoid confusion.